### PR TITLE
Thermal Cultivation compatibility

### PR DESCRIPTION
**Pull request by nbrichau**

### CODE REVIEW

File: mod_thermal/mutations/barley_mutation.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  "enabled": true,
+  "mods": [
+    "thermal",
+    "agricraft",
+    "minecraft"
+  ],
+  "chance": 0.5,
+  "child": "thermal:barley_plant",
+  "parent1": "vanilla:wheat_plant",
+  "parent2": "vanilla:rice_plant"
```
Comment (InfinityRaider) [inline mod_thermal/mutations/barley_mutation.json:14]: Vanilla has a rice plant?

File: mod_thermal/plants/peanut_plant.json [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+      "type": "item",
+      "object": "thermal:peanut_seeds",
+      "useTag": false,
+      "data": "",
+      "ignoredData": [
+        "*"
+      ]
+    }
+  ],
+  "s": [
```
Comment (InfinityRaider) [inline mod_thermal/plants/peanut_plant.json:27]: Why do all of these say "s" instead of "stages"?

File: mod_thermal/plants/rice_plant.json [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+    "seasons": [
+      "spring",
+      "summer",
+      "autumn",
+      "winter"
+    ],
+    "conditions": [],
+    "fluid": {
+      "type": "fluid",
+      "object": "minecraft:empty",
```
Comment (InfinityRaider) [inline mod_thermal/plants/rice_plant.json:93]: Shouldn't this be water?

File: mod_thermal/plants/tea_plant.json [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+    },
+    "soil_nutrients": {
+      "condition": "high",
+      "type": "equal_or_higher",
+      "tolerance_factor": 0.1
+    },
+    "min_light": 10,
+    "max_light": 16,
+    "light_tolerance_factor": 0.5,
+    "seasons": [
```
Comment (InfinityRaider) [inline mod_thermal/plants/tea_plant.json:84]: Seasons are not defined for any of these.

**Review by InfinityRaider — COMMENTED** [review]
Thanks, great work, but some minor modifications are needed I think.

File: mod_thermal/plants/rice_plant.json (hunk 3 above)
Comment (nbrichau) [reply mod_thermal/plants/rice_plant.json:93]: I don't think so, the termal crop doesn't need to be in water

File: mod_thermal/plants/peanut_plant.json (hunk 2 above)
Comment (nbrichau) [reply mod_thermal/plants/peanut_plant.json:27]: that's an error caused when I used the `sed` command and I didn't see it

**Comment by InfinityRaider** [comment]
As always, thanks a ton!

**Pull request merged**